cms/static/js/views/partition_group_list.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This class defines a list view for partition groups.
+ * It is expected to be backed by a Group collection.
+ */
+define([
+    'underscore', 'js/views/list', 'js/views/partition_group_item', 'gettext'
+], function(_, ListView, PartitionGroupItemView, gettext) {
+    'use strict';
+
+    var PartitionGroupListView = ListView.extend({
+        initialize: function(options) {
+            ListView.prototype.initialize.apply(this, [options]);
+            this.scheme = options.scheme;
+        },
+
+        tagName: 'div',
+
+        className: 'partition-group-list',
+
+        // Translators: This refers to a content group that can be linked to a student cohort.
+        itemCategoryDisplayName: gettext('content group'),
```

Comment:
Uck, these "content group" messages aren't very nice in this class. However, I suppose this can be cleaned up if we ever have another editable type of UserPartition/GroupConfiguration.